Assets/LoggingManager/LogStore.cs

```diff
@@ -0,0 +1,276 @@
+﻿using System;
+using System.CodeDom;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Boo.Lang.Runtime;
+using UnityEngine;
+
+
+public enum LogType
+{
+    Normal,
+    Meta
+}
+
+public class LogStore : MonoBehaviour
+{
+
+    private SortedDictionary<string, List<string>> logs;
+    private StringBuilder logString;
+    public string Label { get; set; }
+
+    private LogType logType;
+    private bool logCommonColumns;
```

Comment:
I did this just to make the LogStore class usable in another project if needed, or if someone wants to log a very specific log which doesn't require these columns. 
It is not a problem to log those columns by default, but I thought it was cleaner to have the possibility of not logging those columns.

If you think that every log **should** contains the common columns, let's remove this feature.

Reply:
@Vincent-SD It's also about reducing code complexity (avoiding to increase burden of maintaining code logic). We did not give an option to disable this before, and I dont want to introduce this right now.

If you want, you can keep this boolean internally in logStore and set to `true` by default. But I think for the public API of logstore, we should not expose this, for the time being.

Reply:
@Vincent-SD Actually, let's remove the bool completely. The thing is, with data analysis, we can always filter out data in post-processing - but we cannot magically create missing data. So I'd rather enforce these columns on people - and stand behind our design decisions. (by e.g. documenting in the README that these columns always are logged).

If people really want to avoid logging them, they can just create hacks - none of our own use cases needs this.